include/deal.II/fe/mapping_q_generic.h

```diff
@@ -81,8 +81,54 @@ template <int,int> class MappingQ;
  * a manifold object to all cells that can then also be used by the mapping
  * classes for higher order mappings.
  *
+ * <h4>Behavior along curved boundaries and with different manifolds</h4>
```

Comment:
Either way is fine. You can have the same section heading in the other class, but just refer to the text in the other place.